packages/circuit-ui/components/ListItem/ListItem.tsx

```diff
@@ -0,0 +1,445 @@
+   * If using the `navigation` variant, the chevron icon will be center aligned with this component.
+   */
+  suffix?: ReactNode;
+  /**
+   * Visually mark the list item as selected.
+   */
+  selected?: boolean;
+  /**
+   * Visually and functionally disable the list item.
+   */
+  disabled?: boolean;
+  /**
+   * Function that is called when the list item is clicked.
+   */
+  onClick?: (event: ClickEvent) => void;
+  /**
+   * Link to another part of the application or external page.
+   */
+  href?: string;
+  /**
+   * Additional data that is dispatched with the tracking event.
+   */
+  tracking?: TrackingProps;
+  /**
+   * The ref to the HTML DOM element
+   */
+  ref?: Ref<HTMLDivElement & HTMLAnchorElement & HTMLButtonElement>;
+}
+
+type DivElProps = Omit<HTMLAttributes<HTMLDivElement>, 'prefix' | 'onClick'>;
+type LinkElProps = Omit<
+  AnchorHTMLAttributes<HTMLAnchorElement>,
+  'prefix' | 'onClick'
+>;
+type ButtonElProps = Omit<
+  ButtonHTMLAttributes<HTMLButtonElement>,
+  'prefix' | 'onClick'
+>;
+
+export type ListItemProps = BaseProps &
+  DivElProps &
+  LinkElProps &
+  ButtonElProps;
+
+type InteractiveProps = { isInteractive: boolean };
+
+type StyledListItemProps = Pick<BaseProps, 'selected'> &
+  InteractiveProps &
+  DivElProps &
+  LinkElProps &
+  ButtonElProps;
+
+const baseStyles = ({ theme }: StyleProps) => css`
+  background-color: ${theme.colors.white};
+  color: ${theme.colors.bodyColor};
+  border: ${theme.borderWidth.mega} solid ${theme.colors.n200};
+  border-radius: ${theme.borderRadius.mega};
+  display: flex;
+  align-items: center;
+  padding: ${theme.spacings.kilo} ${theme.spacings.mega};
+  margin: 0;
+  width: 100%;
+  text-align: left;
+  text-decoration: none;
+  position: relative;
+
+  &:focus {
+    border-color: transparent;
+    z-index: 2;
+  }
+
+  &:focus:not(:focus-visible) {
+    border-color: ${theme.colors.n200};
+    z-index: auto;
+  }
+
+  &:disabled,
+  &[disabled] {
+    ${disableVisually()};
+  }
+`;
+
+const interactiveStyles = ({
+  theme,
+  isInteractive,
+}: StyleProps & StyledListItemProps) =>
+  isInteractive &&
+  css`
+    cursor: pointer;
+
+    &:hover {
+      background-color: ${theme.colors.n100};
+    }
+
+    &:active {
+      background-color: ${theme.colors.n200};
+      border-color: ${theme.colors.n200};
+    }
+  `;
+
+const selectedStyles = ({
+  theme,
+  selected,
+}: StyleProps & StyledListItemProps) =>
+  selected &&
+  css`
+    background-color: ${theme.colors.b100};
+
+    &:hover,
+    &:active {
+      background-color: ${theme.colors.b100};
+    }
+
+    &:after {
+      content: '';
+      position: absolute;
+      top: -${theme.borderWidth.mega};
+      bottom: -${theme.borderWidth.mega};
+      left: -${theme.borderWidth.mega};
+      right: -${theme.borderWidth.mega};
+      border: ${theme.borderWidth.mega} solid ${theme.colors.b500};
+      border-radius: ${theme.borderRadius.mega};
+      z-index: 1;
+      pointer-events: none;
+    }
```

Comment:
The primary blues should always be referenced as `pXXX`.

```suggestion
    background-color: ${theme.colors.p100};

    &:hover,
    &:active {
      background-color: ${theme.colors.p100};
    }

    &:after {
      content: '';
      position: absolute;
      top: -${theme.borderWidth.mega};
      bottom: -${theme.borderWidth.mega};
      left: -${theme.borderWidth.mega};
      right: -${theme.borderWidth.mega};
      border: ${theme.borderWidth.mega} solid ${theme.colors.p500};
      border-radius: ${theme.borderRadius.mega};
      z-index: 1;
      pointer-events: none;
    }
```